client/http/client.go

```diff
@@ -76,11 +78,22 @@ func (client *HttpClient) Put(path string, request interface{}, response interfa
 }
 
 func (client *HttpClient) Get(path string, params map[string]string, response interface{}) error {
-	result, err := client.request().
-		SetQueryParams(params).
-		SetResult(response).
-		Get(path)
-	return client.httpResult(result, err)
+	request := client.request().SetQueryParams(params)
+
+	responseType := reflect.TypeOf(response)
+
+	if responseType.Kind() == reflect.Ptr && responseType.Elem().Kind() == reflect.String {
```

Comment:
if string pointer type (golang reflection).

Reply:
Can you elaborate when each type of response it expected?

Reply:
This depends on what the user passes to the function.
If he passes a string pointer - handle special case.
For everything else, use the regular case.

I hope I understood the question. Let me know if it's still unclear.